examples/bri-3/src/bri/workgroup/workgroups/agents/workgroups.agent.ts

```diff
@@ -25,11 +25,11 @@ export class WorkgroupAgent {
       administratorIds,
     );
 
-    if (administratorIds.length != bpiSubjects.length) {
+    if (administratorIds.length !== bpiSubjects.length) {
       throw new NotFoundException(BPISUBJECT_NOT_FOUND_ERR_MESSAGE);
     }
 
-    bpiSubjects.map((bp) => {
+    bpiSubjects.forEach((bp) => {
```

Comment:
just curious, do we have any specific reasons to change this?

Reply:
What immutability is being lost here? I feel like its a memory optimization because there is no need for a return value in this case.

Reply:
map iterates over an input in an immutable manner, while this is not true with forEach.
I'm not against this change, just want to understand why are we making this change in auth cleanup?

Reply:
i was cleaning up linter warnings with this PR too, i have some linter in vs code that was throwing warnings here:
```
Consider using "forEach" instead of "map" as its return value is not being used here.sonarlint(typescript:S2201)
```
can revert since it is not our linter doesnt report it but sonarlint extension i have installed

Reply:
well, map can be used for both the purposes. AFAIK, forEach loses immutability. No issues, we can keep this change.